### DISCUSSION TITLE
Passing a parameter to a triggered job

### DISCUSSION THREAD

**Discussion by yuval-airis** [dagster-io/dagster#18887] [Q&A] [answered]
I would like to trigger a job based on an external call of some sort. I considered either performing that through graphQL (Though [it doesn't work for me yet](https://github.com/dagster-io/dagster/discussions/18885)) or via command line (Through [dagster job command](https://docs.dagster.io/_apidocs/cli#dagster-job)).

In the latter, is it possible for me to add a parameter to the call to indicate the content of the trigger I received?

**Comment by johannkm** [comment, marked as answer]
`dagster job execute` takes a `--tag` parameter that you could pass the info with
